src/Microsoft.Azure.SignalR.Management/ServiceManagerImpl.cs

```diff
@@ -7,10 +7,8 @@
 using System.Security.Claims;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Azure.SignalR.Common.RestClients;
```

Comment:
ad a general practice, if the file only changes format, revert the change to keep the code change PR minimum. (Format change can be in a separate PR)

Reply:
I relocate this file and the namespace is no longer exists, so I think this change relates to the PR.